src/components/screen/__tests__/SearchUser.test.js

```diff
@@ -7,6 +7,8 @@ import { IC_BACK, IC_SEARCH, IC_ICON } from '../../../utils/Icons';
 import renderer from 'react-test-renderer';
 import { shallow, render } from 'enzyme';
 
+jest.useFakeTimers(); // related to Animated Component
```

Comment:
I hope you to use this inside `describe` or `it` statement (where you need it).
I think stating it as global isn't a good idea.

Reply:
I moved this code to place where in the `describe`